src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -238,9 +238,9 @@ export class QueryResultsView extends Disposable {
 				this._panelView.showTab(this.messagesTab.identifier);
 			}
 			// Currently we only need to support visualization options for the first result set.
-			if (runner.batchSets[0]?.resultSetSummaries[0]?.visualization) {
-				const batchSet = runner.batchSets[0];
-				const resultSet = batchSet.resultSetSummaries[0];
+			const batchSet = runner.batchSets[0];
+			const resultSet = batchSet?.resultSetSummaries ? batchSet.resultSetSummaries[0] : undefined;
```

Comment:
Can just do `const resultSet = batchSet?.resultSetSummaries?.[0]`

Reply:
I know there has to be a way! 👍